### DISCUSSION TITLE
v2.0 - New looks for text-generation-webui!

### DISCUSSION THREAD

**Discussion by oobabooga** [oobabooga/text-generation-webui#6586] [Announcements]
https://github.com/oobabooga/text-generation-webui/releases/tag/v2.0

| BEFORE | AFTER |
|--------|------------|
| ![Image1](https://github.com/oobabooga/screenshots/raw/main/BEFORE-INSTRUCT.png) | ![Image1](https://github.com/oobabooga/screenshots/raw/main/AFTER-INSTRUCT.png) |
| ![Image1](https://github.com/oobabooga/screenshots/raw/main/BEFORE-CHAT.png) | ![Image1](https://github.com/oobabooga/screenshots/raw/main/AFTER-CHAT.png) |
| ![Image1](https://github.com/oobabooga/screenshots/raw/main/BEFORE-DEFAULT.png) | ![Image1](https://github.com/oobabooga/screenshots/raw/main/AFTER-DEFAULT.png) |
| ![Image1](https://github.com/oobabooga/screenshots/raw/main/BEFORE-PARAMETERS.png) | ![Image1](https://github.com/oobabooga/screenshots/raw/main/AFTER-PARAMETERS.png) |

**Comment by LeeThompson** [comment]
So the top tabs are now on the side?

**Comment by TheFrankyDoll** [comment]
Awesome job on the redesign! If I were to tweak it a bit, I think top tabs should've stayed on top, multiple left tabs look clunky.

Also, is there any plans to make an editable color scheme?

**Comment by jdbornem** [comment]
I just upgraded, and the multiple collapsing side tabs are really awkward to use. Is there a way to downgrade back to the prior version?
